### DISCUSSION TITLE
Raster image saving to html

### DISCUSSION THREAD

**Discussion by Sudhip007** [banesullivan/localtileserver#173] [Q&A] [answered]
Hi Team,

I have run the raster image with localtileserver and at the end I saved the map into html format. It displayed my raster image, but after closing jupyter notebook and I opened my saved html file then it not displaying my raster image. Please, help me out in this issue.

I used folium for this.

Thanks..

**Comment by banesullivan** [comment, marked as answer]
See https://github.com/banesullivan/localtileserver/issues/174
